src/arcade/patch/agent/process/PatchProcessMetabolismMedium.java

```diff
@@ -61,6 +61,7 @@ public class PatchProcessMetabolismMedium extends PatchProcessMetabolism {
      *   <li>{@code AUTOPHAGY_RATE} = rate of autophagy
      *   <li>{@code ATP_PRODUCTION_RATE} = rate of ATP production
      *   <li>{@code INITIAL_GLUCOSE_CONCENTRATION} = initial cell internal glucose concentration
+     *   <li>{@code INITIAL_GLUCOSE_CONCENTRATION} = initial cell internal glucose concentration
```

Comment:
Remove these duplicates (probably from the merge?)

Reply:
fixed !